tests/sentry/receivers/test_superuser.py

```diff
@@ -0,0 +1,53 @@
+from sentry.auth.superuser import is_active_superuser
+from sentry.models import User
+from sentry.receivers.superuser import disable_superuser, enable_superuser
+from sentry.testutils import TestCase
+
+
+class SuperuserReceiverTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.superuser = User(is_superuser=True, email="test@sentry.io")
```

Comment:
QQ - does the identical email cause any issues at all?

Reply:
i don't believe so since `is_superuser` will be correct for the respective cases but I'm not 100% sure so ill will change it